ipaclient/install/client.py

```diff
@@ -2345,6 +2345,75 @@ def update_ipa_nssdb():
                                    (nickname, sys_db.secdir, e))
 
 
+def sync_time(options, fstore, statestore, force):
+    """
+    Will disable any other time synchronization service if there is
+    --force-chrony option set, and configure chrony with given ntp(chrony)
+    server and/or pool using Augeas in configure_chrony method.
+    If there is no option --ntp-server set IPADiscovery will try to find ntp
+    server in DNS records.
+    """
+    # We assume that NTP servers are discoverable through SRV records in DNS.
+
+    # disable other time&date services first
+    if force:
+        timeconf.force_chrony(statestore)
+
+    print("Synchronizing time")
+    logger.info('Synchronizing time with KDC...')
+
+    if not options.ntp_servers:
+        ds = ipadiscovery.IPADiscovery()
+        ntp_servers = ds.ipadns_search_srv(cli_domain, '_ntp._udp',
+                                           None, break_on_first=False)
+    else:
+        ntp_servers = options.ntp_servers
+
+    if ntp_servers:
+        if timeconf.configure_chrony(ntp_servers, options.ntp_pool,
+                                     fstore, statestore):
+            print("Done Configuring chrony.")
+        else:
+            print("Warning: IPA Server was unable to sync time with chrony!")
```

Comment:
Should the warning mention IPA Server explicitly?

Reply:
Right i should change it.